### ISSUE TITLE
New icons for macOS 11 Big Sur

### ISSUE THREAD

**Issue by elrumo** [iina/iina#3034]
**What you want IINA to do:**

Seeing that macOS 11 Big Sur has introduced a complete redesign of all icons, it might make sense to consider adapting the Iina icon to the new OS when it launches, just to keep it cohesive with the rest of the new design language.]

I've created a couple of variants of the logo following Apple's new icon template. Thoughts?

[<img src="https://github.com/elrumo/macOS-Big-Sur-icons-replacements/blob/master/icons/png/Iina.png?raw=true)" width="100">](https://github.com/elrumo/macOS-Big-Sur-icons-replacements/raw/master/icons/Iina.icns) | [<img src="https://github.com/elrumo/macOS-Big-Sur-icons-replacements/blob/master/icons/png/Iina_alt.png?raw=true)" width="100">](https://github.com/elrumo/macOS-Big-Sur-icons-replacements/raw/master/icons/Iina_alt.icns)
 --- | ---
| [Iina Option 1](https://github.com/elrumo/macOS-Big-Sur-icons-replacements/raw/master/icons/Iina.icns) | [Iina Option 2](https://github.com/elrumo/macOS-Big-Sur-icons-replacements/raw/master/icons/Iina_alt.icns)

**Comment by owenfiscus** [comment]
I vote Option 1, matches the current logo well. Great work!

**Comment by MoNTE48** [comment]
I vote Option 2, prefer simplicity.

**Comment by gferreroferri** [comment]
option 2

**Comment by Zson711** [comment]
option 2 has disappeared？

**Comment by elrumo** [comment]
> option 2 has disappeared？

My bad, I accidentally deleted the image. Should work now.

**Comment by BasO** [comment]
Both really nice. Right now, borderless seems to match Apple's icons better. In beta 2, only some admin apps, like Console and Terminal, have borders.

**Comment by elrumo** [comment]
> Both really nice. We could also use this Big Sur beta period to invite more icon options.
> 
> Concerning border or no border - IMO no border seems to match Apple's icons better. In beta 2, only some admin apps, like Console and Terminal, have borders. On the other hand, without a border the icon is rather low contrast. Maybe it would look better if the logo were a bit brighter.

Would this be better? I think you're right about the contrast, the image bellow shows how the rest of the icon doesn't really stand out. I've made another version with thinner borders, what do you think?

![Iina](https://i.imgur.com/TpQIgvl.png)
![Iina](https://i.imgur.com/rnLYNLD.png)

**Comment by iAYMANi** [comment]
I vote Option 2

**Comment by BasO** [comment]
Here's all 3 in context:

Variant 1:
<img width="1920" alt="Schermafbeelding 2020-07-22 om 15 15 04" src="https://user-images.githubusercontent.com/9433827/88180834-3e0fb480-cc2e-11ea-803c-543b9214ceec.png">

Variant 2:
<img width="1920" alt="Schermafbeelding 2020-07-22 om 15 14 16" src="https://user-images.githubusercontent.com/9433827/88180856-4667ef80-cc2e-11ea-9772-eacca0db1907.png">

Variant 3:
<img width="1920" alt="Schermafbeelding 2020-07-22 om 15 11 57" src="https://user-images.githubusercontent.com/9433827/88180872-4d8efd80-cc2e-11ea-9abc-c2c786bcb72f.png">

> Would [variant 3] be better? I think you're right about the contrast, the image bellow shows how the rest of the icon doesn't really stand out. I've made another version with thinner borders, what do you think?

I still prefer borderless, both for visual coherence and also for following Apple in only leaving some system admin apps with a border (I think these borders imply the app is a sort of system dashboard).

By putting your icons next to some other black-ish icons, I noticed that your icons have a deeper black than the others. I think if you tweak that for variant 2 (and make the dark blue part of the icon a bit lighter, and perhaps the blue should also be a bit lower contrast - not sure), then we'll be basically there. At least IMO.

**Comment by elrumo** [comment]
I've taken the Apple TV icon as reference and added a blue hint to gradient and a subtle white glow to the top of the icon. I've also enlarged the inner elements too. Bellow are the previous icon and the new on and the old one with a dark and a light background for context, thoughts? 

<img src="https://i.imgur.com/ZreLi6i.png" width="100">

With dark background:
<img src="https://i.imgur.com/1F0yvDt.png">

With light background:
<img src="https://i.imgur.com/3dgmUKh.png">

**Comment by BasO** [comment]
And here it is between system app icons:
<img width="1920" alt="Schermafbeelding 2020-07-22 om 21 44 08" src="https://user-images.githubusercontent.com/9433827/88222858-dfb2f800-cc66-11ea-9060-8eaa16b558cb.png">

IMO - the lighter black seems perfect. I prefer no white glow on top (for consistency with the system apps). Concerning logo size, as a sort of objective method, you could use the horizontal border distances in the icons for Mail, FaceTime and Books - they all have the same logo-to-border distance.

Also hoping we can have some other voices in this thread.

**Comment by MadhavChoudhary** [comment]
how about this?

![IINA](https://user-images.githubusercontent.com/29799995/88459263-9e625880-ceb1-11ea-916b-74e354807af3.png)

**Comment by BasO** [comment]
> how about this?

It's not loading for me

**Comment by CineDev** [comment]
Variant 3 with grey border is perfect!

**Comment by elrumo** [comment]
@CineDev Do you mean this one?
 [<img src="https://github.com/elrumo/macOS_Big_Sur_icons_replacements/blob/master/icons/png/low-res/Iina_alt2.png?raw=true" width="100">](https://github.com/elrumo/macOS-Big-Sur-icons-replacements/raw/master/icons/Iina_alt.icns)

**Comment by elrumo** [comment]
I've added the third variant that I made based on the feedback to the original post.

Also, how should we proceed? Should I make a pull request?

**Comment by CineDev** [comment]
@elrumo meant Variant 3 in this post https://github.com/iina/iina/issues/3034#issuecomment-662451090 (just to make a border a little grey). But yours good too. According to new macOS style, maybe it feels even more native.

**Comment by BasO** [comment]
@elrumo option 3 in your OP is not loading, there's no Iina_alt2 in your icns or png folder.

I've used your last icon ([this one](https://github.com/iina/iina/issues/3034#issuecomment-662552896)) for the last couple of weeks in my dock. It's really good. Only thing that I still feel hesitant about is the small glow on top. I want to recommend to remove it; I really believe it's an unnecessary element in the design and in addition it creates this small incongruency with Apple's icons. Can you make an icon without that small glow on top?

After that change, I'm up for proceeding :-)

**Comment by elrumo** [comment]
You're right, there's no Iina_alt2 anymore because I removed it as I thought it was too similar to the other one and didn't add anything new, also, no one really downloaded it.

Already removed the small glow at the top, so feel free to download it and make the PR. You can find the .icns and high res .png files on this .zip. 
[Iina.zip](https://github.com/iina/iina/files/5143123/Iina.zip)

**Comment by BasO** [comment]
@lhc70000 what do you think of this Big Sur style icon? And how should we proceed?

Attached image shows the icon in the [zip from the comment above](https://github.com/iina/iina/files/5143123/Iina.zip) in a semi-default Big Sur context.
<img width="144" alt="Schermafbeelding 2020-08-29 om 14 56 13 (2)" src="https://user-images.githubusercontent.com/9433827/91637322-f4be3c00-ea07-11ea-90f4-fca432f9b71f.png">

**Comment by chrenn** [comment]
Now would probably be a good time to transition to a new logo since pretty much all assets (document icons, etc.) need to be updated for Big Sur anyway. So here's my suggestion based on issue #1782 from two (😅) years ago.

<img width="256" alt="icon" src="https://user-images.githubusercontent.com/30233933/91709044-776b0680-eb82-11ea-92b6-bb413979cb72.png">

Quick reasoning: The logo mimics both a play/pause symbol and letters from the name IINA.

Here's how it looks in the dock. Thoughts?

<img width="256" alt="dock" src="https://user-images.githubusercontent.com/30233933/91711230-dc742b80-eb85-11ea-9100-e2c6163a2a79.png">

**Comment by guohui-ma** [comment]
> Here's all 3 in context:
> 
> Variant 1:
> <img alt="Schermafbeelding 2020-07-22 om 15 15 04" width="1920" src="https://user-images.githubusercontent.com/9433827/88180834-3e0fb480-cc2e-11ea-803c-543b9214ceec.png">
> 
> Variant 2:
> <img alt="Schermafbeelding 2020-07-22 om 15 14 16" width="1920" src="https://user-images.githubusercontent.com/9433827/88180856-4667ef80-cc2e-11ea-9772-eacca0db1907.png">
> 
> Variant 3:
> <img alt="Schermafbeelding 2020-07-22 om 15 11 57" width="1920" src="https://user-images.githubusercontent.com/9433827/88180872-4d8efd80-cc2e-11ea-9abc-c2c786bcb72f.png">
> 
> > Would [variant 3] be better? I think you're right about the contrast, the image bellow shows how the rest of the icon doesn't really stand out. I've made another version with thinner borders, what do you think?
> 
> I still prefer borderless, both for visual coherence and also for following Apple in giving abstract (not real-life based) system app icons a border (I think these borders imply the app is a sort of system dashboard).
> 
> By putting your icons next to some other black-ish icons, I noticed that your icons have a deeper black than the others. I think if you tweak that for variant 2 (and make the dark blue part of the icon a bit lighter, and perhaps the blue should also be just slightly lower contrast - not sure), then we'll be basically there. At least IMO.

I think variant2 is better.

**Comment by BarryMode** [comment]
Variant 2 is the best.

**Comment by lhc70000** [comment]
This is my tentative outcome after trying for a few days:

![iina-icon-512](https://user-images.githubusercontent.com/8478049/95004916-3dc76880-0624-11eb-8820-decf457f27f7.png)

**Comment by Ranoiaetep** [comment]
Definitely great works! However, I do wonder if there's a native solution for apps to fallback to another icon if it was used on an older system.
<img width="500" alt="Screen Shot 2020-10-15 at 3 10 39 PM" src="https://user-images.githubusercontent.com/55516229/96191404-dd280d80-0ef8-11eb-8f61-06fc2e71b3ff.png">
I'm stuck on older machines here. The new square look does look off comparing to other native apps, along with the new Xcode icon. 
Disregard this if it's not possible.

**Comment by BasO** [comment]
@Ranoiaetep There's no native solution, so I would suggest sending this idea to Apple via https://feedbackassistant.apple.com. Who knows.

**Comment by lhc70000** [comment]
@Ranoiaetep Yes, that's exactly what I was considering, but I didn't have enough time to add it in 1.1.0. I'll do this in the next version.

I'm also thinking about allowing users to choose their preferred icons from a community-created icon gallery, so that all previous designs in this thread can be exposed to users.

**Comment by lhc70000** [comment]
@BasO 

Well, at least we can [change the dock icon](https://developer.apple.com/documentation/appkit/nsapplication/1428744-applicationiconimage), although I have never tried it.

**Comment by saagarjha** [comment]
Word on the grapevine is that having one icon across OSes is the intended and desired behavior, and any comparability shims that Apple has put in place during the transition are going away in the future. Feel free to file a feedback about this, but as of now this is what Apple would like us to do.

**Comment by lemikeone** [comment]
> Already removed the small glow at the top, so feel free to download it and make the PR. You can find the .icns and high res .png files on this .zip.
> [Iina.zip](https://github.com/iina/iina/files/5143123/Iina.zip)

This is the best version to my opinion, better than the current official one.

**Comment by hazarek** [comment]
_Document icons_ are differenct from new _App icon_ https://github.com/iina/iina/tree/develop/iina/Assets.xcassets/DocIcons 

Also, the document icons must be updated according to Big Sur Apple guidelines

**Comment by dorjanbenic** [comment]
The icon is great, I like it. Just want to inform you that it is a little bit smaller than all the others in the Dashboard.
![Screen Shot 2021-03-23 at 20 33 07](https://user-images.githubusercontent.com/39588255/112207136-0f136d80-8c17-11eb-84c8-9f21e4d9d491.png)

**Comment by whatsnewsisyphus** [comment]
I just wanted to check in here to see if there is interest for a new, more distinct logo. Here's just a sketch of the idea, the particular lines, colors and angles would be refined and workshopped if people like the general direction.
![iina-01](https://user-images.githubusercontent.com/454772/152191910-26e7b6be-2699-4222-8fb9-c4e07677e87b.png)

**Comment by hazarek** [comment]
@whatsnewsisyphus The logotype you suggested is not related to the topic of this issue.(App icons for macOS)

**Comment by whatsnewsisyphus** [comment]
@hazarek I am aware, however, this thread modified the logo above and it wasn't documented elsewhere so I mentioned it here for interested contributors. I could start a new issue and link here if you prefer

**Comment by hazarek** [comment]
> @hazarek I am aware, however, this thread modified the logo above and it wasn't documented elsewhere so I mentioned it here for interested contributors. I could start a new issue and link here if you prefer

I mean the logotype is not suitable for the icon. A title should be opened on the Discussions page for a logotype proposal. But the owner of this repository (@lhc70000) did not turn on the Discussions feature.

**Comment by lhc70000** [comment]
@hazarek Just opened.

**Comment by whatsnewsisyphus** [comment]
New logo and icon proposal discussion here https://github.com/iina/iina/discussions/3631
